### PR TITLE
Updated model

### PR DESCRIPTION
**Pull request by Auspicus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +11,13 @@
...
   },
   "scripts": {
+    "gulp": "gulp",
     "start": "gulp serve",
     "start:debug": "cross-env DEBUG=pa11y-api:* yarn start",
     "build": "gulp",
     "lint": "esw *.js server config --color",
     "lint:watch": "yarn lint -- --watch",
-    "precommit": "yarn lint && yarn test",
+    "precommit": "npm run lint && npm run test || true",
```
Comment (Auspicus) [inline package.json]: `|| true` lets the lints run but doesn't stop you from committing if they fail

File: server/controllers/issue.controller.js [hunk 2]
```diff
@@ -39,6 +39,7 @@ function create(req, res, next) {
     selector: req.body.selector,
     type: req.body.type,
     typeCode: req.body.typeCode,
+    reportId: req.body.reportId,
```
Comment (Auspicus) [inline server/controllers/issue.controller.js:42]: Added a reference to the report this issue is related to

File: server/controllers/report.controller.js [hunk 3]
```diff
@@ -45,8 +45,9 @@ function create(req, res, next) {
  */
 function update(req, res, next) {
   const report = req.report;
-  report.urls = req.body.urls;
-  report.standard = req.body.standard;
+  report.urls = req.body.urls || report.urls;
```
Comment (Auspicus) [inline server/controllers/report.controller.js:48]: Lets you specify multiple fields to update or only 1 (useful when updating progress)

File: .env.example [hunk 4]
```diff
@@ -1,5 +1,5 @@
-NODE_ENV=development
-PORT=4040
+NODE_ENV=production
```
Comment (rcaracaus) [inline .env.example]: we shouldn't be using htis

**Pull request merged**